torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8770,6 +8685,76 @@ def wrapper(x: np.ndarray, *args, **kwargs):
             SkipInfo('TestReductions', 'test_dim_empty_keepdim'),
         ),
     ),
+    ReductionOpInfo(
+        'argmax',
+        supports_multiple_dims=False,
+        supports_autograd=False,
+        result_dtype=torch.int64,
+        dtypes=all_types_and(torch.float16, torch.bfloat16),
+        ref=reference_reduction_numpy(np.argmax, supports_keepdims=False),
+        skips=(
+            # FIXME: keepdim parameter is ignored when dim=None
+            SkipInfo('TestReductions', 'test_dim_default_keepdim'),
+            SkipInfo('TestReductions', 'test_dim_none_keepdim'),
+        ),
+    ),
+    ReductionOpInfo(
```

Comment:
Maybe we should write a test that enforces an alphabetical order...

Reply:
(But not seriously because we might break up `op_db` into smaller pieces because it's growing too big)